sys/net/application_layer/emcute/emcute.c

```diff
@@ -345,7 +345,6 @@ int emcute_pub(emcute_topic_t *topic, const void *data, size_t len,
     mutex_lock(&txlock);
 
     size_t pos = set_len(tbuf, (len + 6));
```

Comment:
Why is it `len + 6 here`?

Reply:
because the publish header is included in the packet's length ;-).